ceph/ceph_admin/bootstrap.py

```diff
@@ -275,7 +275,7 @@ def bootstrap(self: CephAdmProtocol, config: Dict):
 
         # Todo: This patch is specific to 5.1 release,
         #   should be removed for next 5.x development builds or release.
-        if rhbuild.split("-")[0] in ["5.1", "5.2"]:
+        if rhbuild.split("-")[0] in ["5.1", "5.2", "6.0"]:
```

Comment:
We should not have the flag set for `6.0`... do you have information that mandates the need to use this flag.

Reply:
Hey @psathyan , I am adding this change for BZ https://bugzilla.redhat.com/show_bug.cgi?id=2073030
Post RHCS 6.0 release, when RGW multisite is supported, we will stop requiring deployments to pass --yes-i-know to 'cephadm bootstrap'. When this happens, the reuest in this BZ is to please do not make the deployment fail if --yes-i-know is passed.
The bug requests that --yes-i-know remain a recognized option, even if it doesn't do anything. Hence, I added this flag over here.
Please let me know if any more details are required.

Reply:
Okay, Thanks @psathyan